tests/suites/3_stateful_iceberg/00_rest/00_0000_create_and_show.sh

```diff
@@ -20,4 +20,10 @@ echo "SHOW CREATE CATALOG iceberg_ctl;" | $BENDSQL_CLIENT_CONNECT
 
 echo "SHOW DATABASES IN iceberg_ctl;" | $BENDSQL_CLIENT_CONNECT
 
+echo "SELECT CURRENT_CATALOG();" | $BENDSQL_CLIENT_CONNECT
+
+echo "USE CATALOG `iceberg_ctl`;" | $BENDSQL_CLIENT_CONNECT
```

Comment:
Remove quotes, write as `USE CATALOG iceberg_ctl;` will fix the test.

Reply:
each call ``BENDSQL_CLIENT_CONNECT="bendsql -uroot --host ${QUERY_MYSQL_HANDLER_HOST} --port ${QUERY_HTTP_HANDLER_PORT} --database ${MYSQL_DATABASE} --quote-style=never"`` in this test would create a new connection cause reset session, I think it's the reason this test fail.

And now this should be fixed now .